src/components/ha-labeled-slider.html

```diff
@@ -8,30 +8,42 @@
       :host {
         display: block;
         padding-bottom: 16px;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
       }
 
       .title {
-        margin-bottom: 16px;
+        font-size: 90%;
         opacity: var(--dark-primary-opacity);
+        line-height: 1.2em;
+        padding-top: 0.3em;
+
       }
 
-      iron-icon {
-        float: left;
-        margin-top: 4px;
+      .icon-container {
+        flex: 1 1 50px;
+        text-align: center;
         opacity: var(--dark-secondary-opacity);
+        width: 60px;
+        margin-top: 5px;
       }
 
       .slider-container {
-        margin-left: 24px;
+        flex: 1 1 200px;
       }
-
       ha-paper-slider {
+        width: 100%;
         background-image: var(--ha-slider-background);
+        border-radius: 8px;
       }
-    </style>
 
-    <div class='title'>[[caption]]</div>
-    <iron-icon icon='[[icon]]'></iron-icon>
+    </style>
+    <div class='icon-container'>
```

Comment:
Should we extract this into an element so we can start applying this pattern everywhere?

Reply:
you mean the icon+caption block?
If so we could get rid of <ha-labeled-slider> and set the background in `<ha-paper-slider>`
Then `<more-info light>` can just add the `labeled icon + <ha-paper-slider>` in it's dom, Instead of nesting stuff even deeper.

Reply:
That makes a lot more sense